src/routing/wrappers/Layout.svelte

```diff
@@ -71,4 +72,7 @@
       <Footer />
     </ContentFooter>
   </Content>
+  <AlertNotice toggleKey="dataErrorsWarning">
+    <p>The staging and development instances of GLAM are likely to have data issues and UI bugs.</p>
```

Comment:
I actually think this copy is great. The only thing I'd suggest is maybe being more specific about "data issues" if we can.

Reply:
I suppose if someone is on dev, they may not care about stage, and vice versa. Maybe something like this.

"This dashboard is under active development. Bugs and [data accuracy issues] may exist."

But I'd defer to Rob for the final copy.